### ISSUE TITLE
Cross-linking issues with forum topics and vice versa

### ISSUE THREAD

**Issue by aschrijver** [humanetech-community/humanetech-community-awareness#18]
Need to create links to forum topics in the various README's and create forum topics that have a copy of the Proposal issues from Github in them and a link to GH issue.

**Comment by georgeshk** [comment]
> and create forum topics that have a copy of the Proposal issues from Github in them and a link to GH issue

Warning regarding posting copies of proposals on the forum: I am not in favor of having two copies of the same document in different places. Not only it adds overhead (which we could do without), but importantly it would incite many members to start commenting on the forum in addition to comments on Github. We need to encourage everyone interested in these projects to come to Github instead.

I'd rather propose we post just a one sentence summary and the link to Github once a proposal is finalized and pushed to the repository.

**Comment by aschrijver** [comment]
That is true. I won't maintain the copy on the forum. Instead it has a 'Latest version' link to github. A copy is required for people who are not part of the campaigners team. I want to keep discussions going on in the forum for 2 reasons:

1. To keep attracting new campaigners/members to the project
2. Because github should be focused on work, tasks, and doing things. All the general discussion will be very hard to handle otherwise. This is where the forum is better.

**Comment by georgeshk** [comment]
@aschrijver, I understand you feel the need to attract as much attention as possible, hence opening up multiple avenues.

But it will be an incredible mess (think of the cross-coordination that would involve) if there were to be two avenues for comments and discussions on the same topics/deliverables.

Sorry, but I have to strongly disagree here. Let us keep all detailed discussions on Github.

**Comment by aschrijver** [comment]
I am just saying it is a one-time copy. And one-time creation of a topic per proposal. There is no cross-synchronisation afterwards. These forum topics may be a mess, and they will. A mess you do not want here.

Or are you saying you just want the one Call for Participation topic and nothing more? That will be even messier there, and the forum will forget about this project.
